CMakeLists.txt

```diff
@@ -7,77 +7,84 @@ option(USE_VCPKG "Use vcpkg for dependencies" OFF)
 if (USE_VCPKG)
     set(VCPKG_FEATURE_FLAGS "versions")
 
-    if(DEFINED ENV{VCPKG_ROOT})
+    if (DEFINED ENV{VCPKG_ROOT})
```

Comment:
Note: these changes seem to be the output of clang-format formatting the CMake config. I don't have any objections to this (any automatically-enforced format is better than no format 🤷🏻 ) but I'm open to reverting the formatting if anyone has objections!